ResearchKit/Common/ORKFormItemCell.m

```diff
@@ -604,6 +619,17 @@ - (void)inputValueDidChange {
 }
 
 - (void)answerDidChange {
+#if 1 // *** Medable: PAT-135
+    // see note in cellInit
+    id answer = self.textField.text;
+    
+    if (answer != ORKNullAnswerValue()) {
```

Comment:
What if set the value to "a", then undo?

Reply:
Also, don't we need the length validation here?

Reply:
length validation is done w/in `textField:shouldChangeCharactersInRange:replacementString:` which returns NO in that case and this method (triggered by the "valueChanged" notification) never gets called.

Reply:
> I don't understand the question What if set the value to "a", then undo?: this would be called such that answer evaluates to "a", followed by another invocation in which it evaluates to an empty string, wouldn't it?

Wouldn't it go from `nil` to `a` to `nil` again?

Reply:
No, it’s only ever nil at the very beginning of the screen’s lifetime, and I think it’s immediately set to a default value anyway. Thereafter it’s always a non-nil string, empty or not. So it’d be “” -> “a” -> “”